astrolabe/cli.py

```diff
@@ -506,5 +537,39 @@ def validate_workload_executor(workload_executor, startup_time,
         exit(1)
 
 
+@spec_tests.command('stats')
+@click.pass_context
+def stats(ctx):
+    with open('results.json', 'r') as fp:
+        stats = json.load(fp)
+    with open('events.json', 'r') as fp:
+        events = json.load(fp)
+    
+    import numpy
+    
+    command_events = events['commands']
```

Comment:
Can you refactor this into a function in the `astrolabe.commands` submodule and call it here instead? The `astrolable.cli` submodule should just have the code that parses the command-line input and dispatches the work.

Reply:
Moved.